src/EventEmitterTrait.php

```diff
@@ -22,13 +21,20 @@
 
 trait EventEmitterTrait
 {
-    protected $listeners = [];
-    protected $onceListeners = [];
+    /**
+     * @var array<string, array<int, (callable)>>
+     */
+    protected array $listeners = [];
 
-    public function on($event, callable $listener)
+    /**
+     * @var array<string, array<int, (callable)>>
+     */
+    protected array $onceListeners = [];
```

Comment:
If we add the exact callable signature to the interface we need to update it he as  well